src/WordPress/Hooks.php

```diff
@@ -250,4 +252,11 @@ public function http2ServerPushInit()
     {
         HTTP2ServerPush::init();
     }
+
+    public function getCloudflareRequestJSON()
+    {
+        if ($_GET['action'] === 'cloudflare_proxy') {
```

Comment:
All HTTP verbs hit the same URL with the `action` set because thats how WordPress determines which AJAX handler to use.